xlive/Blam/Engine/game/game_engine_util.cpp

```diff
@@ -17,11 +17,11 @@ void game_engine_check_for_round_winner()
 	p_game_engine_check_for_round_winner();
 }
 
-void game_engine_end_round_with_winner(DWORD player_index, bool unk_bool)
+void game_engine_end_round_with_winner(DWORD player_index, bool go_to_next_round)
```

Comment:
in the end what is this first parameter, `player_index` or `player_team`?

Reply:
it's actually both, i'll update the parameter name

Reply:
changed to player_datum_or_team_index